src/microbe_stage/MicrobeCamera.tscn

```diff
@@ -48,12 +72,51 @@ current = true
 fov = 90.0
 script = ExtResource("3")
 DefaultCameraHeight = 30.0
+blurPlanePath = NodePath("BackgroundPlane")
+blurColorRectPath = NodePath("SubViewport2/ColorRect")
+backgroundPlanePath = NodePath("SubViewport/BackgroundPlane")
+
+[node name="AudioListener3D" type="AudioListener3D" parent="."]
+current = true
 
 [node name="BackgroundPlane" type="CSGMesh3D" parent="."]
-transform = Transform3D(1, 0, 0, 0, -1.62921e-07, -1, 0, 1, -1.62921e-07, 0, 5.96247e-08, -15)
+transform = Transform3D(1, 0, 0, 0, -1.62921e-07, -1, 0, 1, -1.62921e-07, 0, 0, -20)
+cast_shadow = 0
+mesh = SubResource("1")
+material = SubResource("ShaderMaterial_1sr76")
+
+[node name="SubViewport2" type="SubViewport" parent="."]
```

Comment:
Could this be implemented as a second render pass or directly baked into the background shader? I ask because we'll hit a rendering performance problem at some point if we just throw subviewports at any problem.

Reply:
Actually, it seems possible to re-implement this without viewports. I've just found out that it's possible for a shader to get it's texture from a CanvasLayer instead of a viewport ([the docs](https://docs.godotengine.org/en/stable/tutorials/shaders/custom_postprocessing.html#multi-pass-post-processing)), which should be more performant.

Reply:
In case that's not usable, I think putting the blur code directly into the background shader would be the right way to code. That would also quite neatly I think expose how expensive in terms of texture reads it is actually to perform a blur.

Reply:
Putting the blur code into the background shader would be bad for performance, because it would then need n*n texture reads (for blur diameter of n), and for each of those reads it would also need to calculate distortion.
The current implementation does blur in 2 passes (vertical and horizontal), and because of that only needs 2*n reads. This difference in calculation costs should probably compensate for viewports' performance.

Reply:
Still it will now blur the moving bubble layers that are separate from the furthest background layer as well, which is probably not the best. Also it could maybe be possible to use a cheaper blur method combined into the distortion math to do a reasonable enough blur with high performance at the same time as applying the distortion math?

I guess I'll let you investigate first what is actually possible to do before dumping more ideas.

Reply:
Apparently the way of making multi-pass shaders with CanvasLayers requires reading the screen texture, so there still needs to be at least one viewport. 
The background plane can also be optimized into a ColorRect (which needs a scale argument passed to it, but it's probably still more performant than rendering 3D geometry).

This is what the resulting scene hierarchy looks like:
![image](https://github.com/user-attachments/assets/295cffe3-ea47-4b11-a2a4-81f0d99b477c)